torch/onnx/_internal/fx/patcher.py

```diff
@@ -77,6 +91,33 @@ def _rebuild_real_tensor(storage, storage_offset, size, stride):
         self.torch_load_wrapper = torch_load_wrapper
         self.torch__util_rebuild_tensor_wrapper = torch__util__rebuild_tensor_wrapper
 
+        if has_safetensors_and_transformers:
+
+            def safetensors_load_file_wrapper(filename, device="cpu"):
```

Comment:
We can follow up with adjustment according to https://github.com/pytorch/pytorch/issues/106732.
I'm ok with this PR as the patch to unblock.